crypto/src/symkey/mod.rs

```diff
@@ -31,7 +31,7 @@ pub enum Error {
     DecryptionError(String),
 }
 
-use self::chacha20poly1305::Chacha20poly1305Key;
+use self::chacha20poly1305::{Chacha20poly1305Key, KEY_LEN};
```

Comment:
This is also unnecessary. When decoding, the decode process will enforce having the correct key length.